com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+<templates>
+	<template name="OpenApi version" 
```

Comment:
"OpenApi" -> "OpenAPI", see [The Specification Name and Usage](https://www.openapis.org/faq/style-guide)